### PR TITLE
Revised Orders and Shipments articles based CM's review

### PR DESCRIPTION
**Pull request by justinchoi001**
https://issues.liferay.com/browse/LRDOCS-7754

### CODE REVIEW

File: docs/commerce/2.x/en/orders-and-fulfillment/shipments/shipments-management-reference-guide.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Shipments Management Reference Guide
```
Comment (jrhoun) [inline docs/commerce/2.x/en/orders-and-fulfillment/shipments/shipments-management-reference-guide.md:1]: Is this article part of Corbin's feedback, @justinchoi001 ?

Comment (jrhoun) [reply docs/commerce/2.x/en/orders-and-fulfillment/shipments/shipments-management-reference-guide.md:1]: Or more directly, is this article supposed to be here?

Comment (justinchoi001) [reply docs/commerce/2.x/en/orders-and-fulfillment/shipments/shipments-management-reference-guide.md:1]: Yes, this is supposed to be here.

File: site/build_site.sh [hunk 2]
```diff
@@ -16,6 +16,21 @@ function check_utils {
 	done
```
Comment (jrhoun) [inline site/build_site.sh:16]: These changes look like they were included in error.

Comment (jrhoun) [reply site/build_site.sh:16]: Actually, may have spoken too soon, this might just be the diff w/ current master.

**Comment by jrhoun** [comment]
Merged. Thanks. Please review my changes @justinchoi001 - particularly around admonition usage, some minor grammar issues (missing period), and having to add the reference guide to the left nav in the RST. https://github.com/liferay/liferay-learn/commit/97fca3cbd8220edebc8e22f0a84c21f3a0b7b99a

**Pull request closed without merging**